### PR TITLE
Clean up BuildFiles under FWCore/

### PR DESCRIPTION
**Pull request by fwyzard**
Clean up BuildFile.xml filea with `xmllint --format`.

### CODE REVIEW

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by fwyzard** [comment]
@cmsbuild, please test

File: FWCore/Skeletons/scripts/mkTemplates/EDProducer/BuildFile.xml [hunk 1]
```diff
@@ -4,5 +4,5 @@
 @example_myparticle <use name="PhysicsTools/UtilAlgos"/>
 @example_myparticle <use name="DataFormats/MuonReco"/>
 @example_myparticle <use name="DataFormats/EgammaCandidates"/>
-@example_myparticle <use name "DataFormats/Candidate"/>
```
Comment (silviodonato) [inline FWCore/Skeletons/scripts/mkTemplates/EDProducer/BuildFile.xml:7]: @fwyzard

**Comment by silviodonato** [comment]
merge

**Comment by makortel** [comment]
+1

post-merge

**Pull request merged**